content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md

```diff
@@ -13,6 +13,12 @@ To avoid CNI plugin-related errors, verify that you are using or upgrading to a
 container runtime that has been tested to work correctly with your version of
 Kubernetes.
 
+{{<note>}} 
+
+From Kubernetes 1.24,the kubelet always uses CNI mode for networking since the legacy `--network-plugin`
+command line parameter has been removed from the binary. For more information, refer to [Troubleshooting Clusters](https://kubernetes.io/docs/tasks/debug/debug-cluster/) page.
+
+{{</note>}}
```

Comment:
This is too early in the page. Make it normal text (not a note callout) and give it a heading.